src/ncp/spinel.h

```diff
@@ -532,7 +532,12 @@ typedef enum
     /** Format: `A(T(ESLCcCbLL))`
      *  eui64, rloc16, age, inLqi ,aveRSS, mode, isChild. linkFrameCounter, mleCounter
      */
-    SPINEL_PROP_THREAD_NEIGHBOR_TABLE  = SPINEL_PROP_THREAD_EXT__BEGIN + 11,
+    SPINEL_PROP_THREAD_NEIGHBOR_TABLE   = SPINEL_PROP_THREAD_EXT__BEGIN + 11,
```

Comment:
Yes. Indeed. The `SPINEL_PROP_THREAD_ROUTER_SELECTION_JITTER` and `SPINEL_PROP_THREAD_PREFERRED_ROUTER_ID` are not aligned correctly.
But the `SPINEL_PROP_THREAD_NEIGHBOR_TABLE` seems correct. So let's keep it as is in this commit/PR. For the other two I will submit a separate (style only) commit/PR.